Packs/IntSight/Integrations/Rapid7ThreatCommand/Rapid7ThreatCommand.yml

```diff
@@ -2231,13 +2231,13 @@ script:
       required: false
       defaultValue: '600'
     outputs:
-    - contextPath: File.md5
+    - contextPath: File.MD5
```

Comment:
1. i added a breaking changes note for changing the outputs name, but it no need for adding a new output, isn't it?
2. no need to make changes on the code side for the changed names, since when the object is created in Common, its created with uppercase letters, so it takes the values returned from the response and map them to how we expect the output to be, i followed the flow, it works as expected.
same goes for adding the `Domain.Name`, we build an indicator of domain type, which automatically adds `Domain.Name` to context, so it's already implemented, but not printed out in context.

Reply:
Ok, so in this case please make sure to say you're not removing/adding an output, but correcting the output to the actual output.